charts/test-guide/templates/deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
                   - "sh"
                   - "-c"
                   - >
-                    cp /app/TTS-TM/licenseV2_temp.ser /app/TTS-TM/licenseV2.ser;
+                    cp {{ .Values.baseSettings.tgWorkspace }}/TTS-TM/licenseV2_temp.ser {{ .Values.baseSettings.tgWorkspace }}/TTS-TM/licenseV2.ser;
```

Comment:
the value contains a trailing slash - remove preceding slash of TTS-TM directory

Reply:
remarks implemented